docs/source/methods/energy_deposition.rst

```diff
@@ -0,0 +1,134 @@
+.. _methods_heating:
+
+=============================
+Heating and Energy Deposition
+=============================
+
+As particles traverse a problem, some portion of their energy is deposited at 
+collision sites. There are a variety of mechanisms that contribute to the
+energy deposition, including down-scattering from higher to lower energies,
+fission events, and reactions with positive Q-values. The information describing
+how much energy is deposited for a specific reaction is referred to as 
+"heating numbers" and can be computed using a program like NJOY with the
+``heatr`` module.
+
+These heating numbers are the product of reaction-specific coefficients and
+a reaction cross section
+
+.. math::
+
+    H(E) = \phi(E)\sum_i\rho_i\sum_rk_{i, r}
```

Comment:
The terminology here is slightly off. "Heating number" refers to the energy deposition per reaction, i.e. the (E + Q - Eout) term. "KERMA" is the heating number times the cross section. The H(E) that you have here is the total heating rate.